x-pack/plugins/infra/public/pages/metrics/inventory_view/components/waffle/waffle_inventory_switcher.tsx

```diff
@@ -31,7 +31,7 @@ export const WaffleInventorySwitcher: React.FC = () => {
   } = useWaffleOptionsContext();
   const [isOpen, setIsOpen] = useState(false);
   const closePopover = useCallback(() => setIsOpen(false), []);
-  const openPopover = useCallback(() => setIsOpen(true), []);
+  const togglePopover = useCallback(() => setIsOpen((currentIsOpen) => !currentIsOpen), []);
```

Comment:
Just FYI, there is a custom hook that could be used here. ex:

```ts
 const [isPopoverOpen, { off: closePopover, toggle: togglePopover }] = useBoolean(false);
```

Mentioning it for the future, if you come across anything that might be interesting to use it.

Reply:
Nice, thank you! 👍